packages/satellite/rubygem-foreman_theme_satellite/rubygem-foreman_theme_satellite.spec

```diff
@@ -0,0 +1,91 @@
+# template: foreman_plugin
+%global gem_name foreman_theme_satellite
+%global plugin_name theme_satellite
+%global foreman_min_version 3.7
+
+Name: rubygem-%{gem_name}
+Version: 13.1.0
+Release: 1%{?foremandist}%{?dist}
+Summary: This is a plugin that enables building a theme for Foreman
+License: GPLv3
+URL: https://github.com/RedHatSatellite/foreman_theme_satellite
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: foreman >= %{foreman_min_version}
+BuildRequires: foreman-assets >= %{foreman_min_version}
+BuildRequires: foreman-plugin >= %{foreman_min_version}
+Requires: ruby >= 2.7
+BuildRequires: ruby >= 2.7
+BuildRequires: rubygems-devel
+BuildArch: noarch
+Provides: foreman-plugin-%{plugin_name} = %{version}
+BuildRequires: rubygem(activesupport)
+BuildRequires: rubygem(deface)
+# end specfile generated dependencies
+
+# start package.json devDependencies BuildRequires
```

Comment:
I'd at least expect `@theforeman/builder` to be here. I think that now has a strict dependency on an older version (`^4.12.0`) so perhaps that's also an issue that needs to be addressed.

Reply:
let me regen the spec and see what happens